NovelRT.Sdk/TemplateFiles/conanfile.py

```diff
@@ -1,27 +1,20 @@
 from conans import ConanFile, CMake
-import os
 
 class ###PROJECT_NAME###Conan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake_find_package", "cmake_paths"
+    generators = "cmake_paths", "cmake_find_package"
     requires = [
-        ("NovelRT/###NOVELRT_VERSION###")]
-    default_options = {
-        "NovelRT:documentation": False,
-        "NovelRT:buildtests":False,
-        "NovelRT:buildsamples":False,
-        "NovelRT:packagemode":True
-    }
+        ("novelrt/###NOVELRT_VERSION###")
+    ]
 
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
 
     def imports(self):
-        self.copy("*.dll", dst="bin", src="bin")
-
-    def test(self):
-        os.chdir("bin")
-        self.run(".%sexample" % os.sep)
+        if self.settings.os == "Windows":
+            self.copy("*.dll", "../deps", "bin")
+        self.copy("*.spv", "../Resources/Shaders", "bin/Resources/Shaders")
+ 
  
```

Comment:
You're missing an EOF here.

Reply:
Can you confirm if this is resolved on your end?